sharepoint/sharepoint-ps/sharepoint-online/Set-SPODisableSpacesActivation.md

```diff
@@ -0,0 +1,139 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://docs.microsoft.com/powershell/module/sharepoint-online/set-spodisablespacesactivation
+applicable: SharePoint Online
+title: Set-SPODisableSpacesActivation
+schema: 2.0.0
+author: trent-green
+ms.author: trgreen
+ms.reviewer:
+---
+
+# Set-SPODisableSpacesActivation
+
+## SYNOPSIS
+Disables the SharePoint Space (Preview) activation.
+
+## SYNTAX
+
+```
+Set-SPODisableSpacesActivation [-Disable] <Boolean> [-Scope] <String> [[-Identity] <SpoSitePipeBind>] [-WhatIf]
+ [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Disables the SharePoint Space (Preview) activation either at Tenant level or Site level.
```

Comment:
Isn't Spaces GA now as of ~month ago?

Reply:
I read _preview_ in another Ms doc I can not find now, but if you have confirmation, I'll remove the preview tag. Thanks